### PR TITLE
Optimize intro sections in schema guide

### PR DESCRIPTION
**Pull request by sdruskat**
**Related issues**

Refs: #206

Includes: #261, #262, #266

**Describe the changes made in this pull request**

- Copy edit some wording
- Rename and edit ex-"transitive credit" section (see also #267)
- Skew credit redirection section slightly towards better practice (of citing both software and paper rather than just paper)
- Fix linting error in schema.json

**Instructions to review the pull request**

- Look at the changes in `schema-guide.md`

### CODE REVIEW

**Comment by sdruskat** [comment]
If 95b61b008e3843480b9b29e6d36c999ecbae9737 goes through, this should be pulled over to **cffinit** as well (ping https://github.com/citation-file-format/cffinit/issues/10).

File: schema-guide.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ title: My Research Software
 
 ### Typical example
 
-For most software however, it is relatively easy to expand the minimal case with some more information like the version, the date when it was last published, some keywords, etc.:
+For most software however, it is relatively easy to expand the minimal case with some more important information like the version, the date when it was last published, some keywords, etc.:
```
Comment (jspaaks) [inline schema-guide.md:26]: a bit ambiguous: do you mean "more important" or "more, important" either way I don;t think this addition is needed

File: schema-guide.md [hunk 2]
```diff
@@ -50,11 +50,15 @@ title: My Research Software
 version: 0.11.2
 ```
 
-### Transitive credit
+### Referencing other work
 
 When your software or data builds on what others have already done, it is good practice to add a `references` section to your
-`CITATION.cff` file. This way, whenever your work gets credited, a little bit of that goes into crediting the works that
-you built on.
+`CITATION.cff` file. This way, you give credit to those on whose work you build your own.
```
Comment (jspaaks) [inline schema-guide.md:56]: yea I struggled with this one as well in the designs of the revamped cffinit website.

maybe something like
```suggestion
`CITATION.cff` file. This way, you give credit to other works on which your own work is built.
```

File: schema-guide.md (hunk 1 above)
Comment (sdruskat) [reply schema-guide.md:26]: Okay, I see, was trying to stress, the importance of having this metadata. Will give this another shot, and if this fails, remove.

File: schema-guide.md (hunk 2 above)
Comment (sdruskat) [reply schema-guide.md:56]: Hm, you don't credit works though, but (ideally) people. Also I've realized that both examples suggest that your own work is *completely* built on other work, which seems to neglect own creativity in building a work. How about this:

```suggestion
`CITATION.cff` file. This way, you give credit to the authors of works that your own work builds on.
```

This way you credit the authors, and also "your own work builds on" suggests that you add a lego piece on top of an existing tower?

Comment (jspaaks) [reply schema-guide.md:56]: works for me, I'll commit the suggestion

**Comment by jspaaks** [comment]
I'll approve the PR, the remaining issue is not so important that it should block further progress

**Review by jspaaks — APPROVED** [review]

**Comment by sdruskat** [comment]
> I'll approve the PR, the remaining issue is not so important that it should block further progress

Thanks, I'll see to how to fix these best, then merge.

File: schema-guide.md (hunk 1 above)
Comment (sdruskat) [reply schema-guide.md]: Done in https://github.com/citation-file-format/citation-file-format/pull/270/commits/3c3ec0c9f4ac515df99a24e5ae3154103dcbbc88.

**Pull request merged**